modulefiles/RDAS/hera.intel.lua

```diff
@@ -86,8 +86,8 @@ setenv('MPIEXEC_NPROC', mpinproc)
 setenv("CRTM_FIX","/scratch1/NCEPDEV/da/Cory.R.Martin/RDASApp/fix/crtm/2.4.0")
 setenv("RDASAPP_TESTDATA","/scratch1/NCEPDEV/da/Cory.R.Martin/CI/RDASApp/data")
 --setenv("RDAS_RRFS_DATA_ROOT","/scratch2/NCEPDEV/fv3-cam/Ting.Lei/dr-emc-rdas/")
---setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Xiaoyan.Zhang/RRFS")
-setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Samuel.Degelia/RRFS")
+setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Xiaoyan.Zhang/RRFS")
+--setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Samuel.Degelia/RRFS")
```

Comment:
why do we keep this commented out line here?  For easier switch back in the future?

Reply:
> why do we keep this commented out line here? For easier switch back in the future?

Yes. The test data is in my directory now. But later, Sam will move them together to his directory. But I can remove the commented out line of your directory if you do not need to keep it.